### PR TITLE
Fix rewrite_urls issue

### LINKED ISSUE #467: Semantic error in HTMLRewriter example
See https://community.cloudflare.com/t/worker-script-to-rewrite-body-html/123091

### PR DESCRIPTION
**Pull request by signalnerve**
Thanks @jkpe for the initial submission! I've updated the snippet without the syntax/linter changes in #466, to make a cleaner diff.

Closes #467

### CODE REVIEW

**Comment by victoriabernard92** [comment]
handleRequest should always be at the top of snippets with help functions at bottom

File: data/snippet/rewrite_urls.toml [hunk 1]
```diff
@@ -27,7 +18,16 @@ class AttributeRewriter {
   }
 }
```
Comment (victoriabernard92) [inline data/snippet/rewrite_urls.toml]: Does this work? I am pretty sure it will break with the newlines here

Comment (signalnerve) [reply data/snippet/rewrite_urls.toml]: What do you mean by work? Like render in the UI? I'm able to render it in the UI, and this code has been tested too!

**Comment by signalnerve** [comment]
Moved `handleRequest`, I think your newline question was around the first part of the snippet code, right? Let me know if I didn't catch it, just pushed a new commit

**Comment by victoriabernard92** [comment]
still see newlines in there but you are right it does work ( i thought \n\n break hugo) Either way i think we should remove any `\n\n` from the code

Also could we add to the description to be more clear what its doing. 

- [ ] Remove \n\n
- [ ] Remove any hard set constants and put them at the top in all caps e.g. `developer.mozilla.org` should be a constant
- [ ] change the title and ID to be more clear. maybe `Rewrite links in HTML` and `rewrite_links_html` ?

**Comment by ashleymichal** [comment]
> handleRequest should always be at the top of snippets with help functions at bottom

we should be careful with this rule; iirc, there was a problem recently with a worker that used a class declaration but having the class definition _after_ the handler that called it was not valid (classes are not hoisted the way functions are, apparently).

**Comment by victoriabernard92** [comment]
closing because tracked at https://github.com/victoriabernard92/template-registry/issues/10

**Pull request closed without merging**